src/components/styled-atlaskit/Dropdown/DropdownMenu.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 AccelByte Inc. All Rights Reserved.
+ * This is licensed software from AccelByte Inc, for limitations
+ * and restrictions contact your company contract manager.
+ */
+
+import React from "react";
+import classNames from "classnames";
+import "./DropdownMenu.scss";
+import { default as AKDropdown } from "@atlaskit/dropdown-menu";
+import { DropdownMenuStatefulProps } from "@atlaskit/dropdown-menu/types";
+import { ButtonAppearances } from "@atlaskit/button";
+
+type DropdownAppearance = Extract<ButtonAppearances, "subtle" | "primary">;
+
+export interface DropDownMenuProps extends DropdownMenuStatefulProps {
+  /** Set if the dropdown menu button is disabled. */
+  isDisable?: boolean;
```

Comment:
i suggest to change this to `isDisabled`, then we can omit it from line 41 as it'll be included in `...props`

Reply:
I think it cannot be included in `...props` since the dropdown-menu from atlaskit do not accept the `isDisabled` props. it only received it via `triggerButtonProps`